src/EPR.Calculator.Frontend/Controllers/CalculationRunNameController.cs

```diff
@@ -33,13 +37,36 @@ public ViewResult Confirmation()
             return this.View(ViewNames.CalculationRunConfirmation);
         }
 
-        private static ErrorViewModel CreateErrorViewModel()
+        private static ValidationResult ValidateCalculationRunName(string calculationName)
```

Comment:
Can you use FluentValidator? This is a standard across the Defra. I can help you with setting up if you need. Thanks

Reply:
@udaydefra, I have updated to FluentValidator.

Reply:
Looks good, Thanks